NuRadioReco/modules/trigger/envelopeTrigger.py

```diff
@@ -91,10 +91,9 @@ def run(self, evt, station, det, passband, order, threshold, coinc_window, numbe
 
             f = np.zeros_like(frequencies, dtype=np.complex)
             mask = frequencies > 0
-            b, a = scipy.signal.butter(order, passband, 'bandpass', analog=True)
-            w, h = scipy.signal.freqs(b, a, frequencies[mask])
+            b, a = scipy.signal.butter(order, passband, 'bandpass', analog=True, output='ba')  #Numerator (b) and denominator (a) polynomials of the IIR filter
```

Comment:
What is this change for?

Reply:
I just wrote down the default, so that the output is b (numerator) and a (polynomials of the IRR). For me, that wasnt obvious so i made a comment. I can delete it, if it is missleading.

Reply:
I mean the output='ba' parameter